wikipedia-dark.user.css

```
@@ -6546,6 +6546,49 @@ regexp("^https://(foundation|donate)\.wikimedia\.org.*$") {
   .list-switcher-element a[role="button"] {
     color: var(--base-color) !important;
   }
+  *[style*="background:#c0cfe4"],
+  *[style*="background:#9999DF"] {
+    background-color: var(--blue-4) !important;
+  }
+  *[style*="background:#a0ade3"],
+  *[style*="background:#ccccff"] {
+    background-color: var(--blue-3) !important;
+  }
+  *[style*="background:#e2e4c0"],
+  *[style*="background:#d0cfa4"] {
+    background-color: var(--brown-2) !important;
+  }
+  *[style*="background:#c0d8e4"] {
+    background-color: var(--blue-3) !important;
+  }
+  *[style*="background:#e0e0b0"] {
+    background-color: var(--pink-1) !important;
+  }
+  *[style*="background:#c0e4c0"],
+  *[style*="background:#99cc99"],
+  *[style*="background:#99CC99"] {
+    background-color: var(--green-3) !important;
+  }
+  *[style*="background:#cfedcc"] {
+    background-color: var(--green-4) !important;
+  }
+  *[style*="background:#e4d4c0"],
+  *[style*="background:#f2caa4"] {
+    background-color: var(--red-3) !important;
```

```suggestion
    background-color: var(--something-not-red) !important;
```

e.g. dark orange-ish `#4d2b01`